ps_client_frontend/src/components/navbar/Navbar.jsx

```diff
@@ -25,6 +27,11 @@ export default function Navbar() {
           <div className="button" onClick={handleLogout}>
             Logout
           </div>
+          <div className="align">
+            <Link to="/profile">
+              <RoundBtn icon={gearIcon} />
+            </Link>
+          </div>
```

Comment:
there is a console.log left behind on line 13. I think I left it honestly but if you could remove it in this pr that would be awesome.